__tests__/app.test.js

```diff
@@ -127,6 +127,140 @@ describe("GET Endpoints", () => {
         });
     });
   });
+
+  describe("GET /api/articles (10) filtered by queries", () => {
+    it("200: Returns an array of objects containing articles filtered by topic > cats", () => {
+      return request(app)
+        .get("/api/articles?topic=cats") // has one cat article
+        .expect(200)
+        .then((response) => {
+          const { articles } = response.body;
+          expect(articles).toBeInstanceOf(Array);
+          expect(articles).toHaveLength(1); // 1 cat article
+          expect(articles).toBeSorted({ key: "created_at", descending: true }); //default
+          articles.forEach((article) => {
+            expect(article.author).toBe('rogersop');
+            expect(article.topic).toBe("cats");
+            expect(article).toMatchObject({
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              topic: expect.any(String),
```

Comment:
I would say we know that its going to be a topic of 'cats'

Reply:
```expect(article.author).toBe('rogersop');
            expect(article.topic).toBe("cats");
```